### PR TITLE
Add release notes for 0.288.1

### PR DESCRIPTION
**Pull request by wanglinsong**
## Description
Add release notes for 0.288.1

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.288.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 Release 0.288
 =============
 
+.. warning::
+
+   Package presto-server-0.288.tar.gz does not have presto-ui module, Presto Console does not work.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/release/release-0.288.rst:7]:
```suggestion
   Package presto-server-0.288.tar.gz does not have presto-ui module, Presto Console does not work. :issue:`23327`
```
Add issue number.

File: presto-docs/src/main/sphinx/release/release-0.288.1.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.288.1
+===============
+
+General Changes
+_______________
+* Rebuild package presto-server to include presto-ui module.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/release/release-0.288.1.rst:7]:
```suggestion
* Rebuild package presto-server to include presto-ui module. :issue:`23327`
```

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the doc! Everything builds and looks good. Suggest adding a link to the issue.

File: presto-docs/src/main/sphinx/release/release-0.288.rst (hunk 1 above)
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.288.rst:7]:
```suggestion
   The tarball package ``presto-server-0.288.tar.gz`` does not include the Presto Console.  This has been fixed in the patch version ``0.288.1``. :issue:`23327`
```

File: presto-docs/src/main/sphinx/release/release-0.288.1.rst (hunk 2 above)
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.288.1.rst:7]:
```suggestion
* The Presto Console is included in this release. :issue:`23327`
```

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

**Review by steveburnett — APPROVED** [review]
LGTM! (docs) 

Pull updated branch, new local doc build. Thanks!

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**